test/testUtils/mayaUtils.py

```diff
@@ -276,6 +277,14 @@ def activeModelPanel():
         if cmds.modelEditor(panel, q=1, av=1):
             return panel
 
+def setBasicCamera(offset=BASIC_CAMERA_DEFAULT_OFFSET):
+    """
+    Sets the camera in a similar position and rotation to the default ones.
+    An offset can be specified to change the distance of the camera from the origin.
+    """
+    cmds.setAttr('persp.translate', offset, .75 * offset, offset, type='float3')
+    cmds.setAttr('persp.rotate', -30, 45, 0, type='float3')
```

Comment:
This method is essentially the same as `setBasicCam` from `mtohUtils`, but AFAIK Mtoh is essentially outdated in this repo, so I ported it here. Note that I did change the default value from 24 to 28 though, as that seems like the default value when creating a new scene.